elastic-utils-lib/src/cmr/elastic_utils/datetime_helper.clj

```diff
@@ -1,32 +1,32 @@
 (ns cmr.elastic-utils.datetime-helper
   "Contains helper functions for converting date time string to elastic date time string"
   (:require
-   [clj-time.format :as f]
-   [clojure.string :as s]
-   [cmr.common.date-time-parser :as p]))
+   [clj-time.format :as time-format]
+   [clojure.string :as string]
```

Comment:
I thought we were going to use a configuration file to enforce alias names? Reasoning being, if it doesn't throw warnings with the linter, folks aren't going to do it.

Reply:
we have a lot of string out there. I thought in this case we wanted the longer one?

Reply:
there are 126 :as str
349 :as string

Reply:
Thomas and I discussed this issue and we've elected to leave this as is for now and address alias standards in a separate ticket